### PR TITLE
Set default quotas for Octavia

### PR DESCRIPTION
**Pull request by gthiemonge**

### CODE REVIEW

**Comment by beagles** [comment]
/test octavia-operator-build-deploy-kuttl

neutron-db-sync is failing with imagepullbackoff

File: controllers/octavia_controller.go [hunk 1]
```diff
@@ -599,6 +599,10 @@ func (r *OctaviaReconciler) reconcileNormal(ctx context.Context, instance *octav
 	// Amphora reconciliation
 	// ------------------------------------------------------------------------------------------------------------
 
+	if err = octavia.EnsureQuotas(ctx, instance, &r.Log, helper); err != nil {
+		return ctrl.Result{}, err
+	}
```
Comment (abays) [inline controllers/octavia_controller.go:604]: A suggestion for a follow-up PR might be to add conditions to house and surface the error here and also on L617.  Maybe we could reuse the condition from L625 (`amphoraControllerReadyCondition(octaviav1.HealthManager)`)?  That way the user could see the problem in `oc get octavia` without having to search through `oc logs <Octavia controller manager pod>`.

**Comment by beagles** [comment]
Patch is self-contained and works in independent verification.  Merging.

**Pull request merged**